tree_test.go

```diff
@@ -1982,3 +1982,69 @@ func Benchmark_GetByIndex(b *testing.B) {
 		}
 	})
 }
+
+func TestNodeCacheStatisic(b *testing.T) {
```

Comment:
nit: suggest changing to `t` since it is idiomatic to name benchmarks as `b` but regular tests as `t`

```suggestion
func TestNodeCacheStatisic(t *testing.T) {
```